efficientdet/third_party/graph_edit/tests/edit_test.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Google Research. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for tensorflow.contrib.graph_editor."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import third_party.graph_edit as ge
+from third_party.graph_edit.tests import match
+from tensorflow.python.framework import constant_op
+from tensorflow.python.framework import ops
+from tensorflow.python.ops import math_ops
+from tensorflow.python.platform import test
+
+
+class EditTest(test.TestCase):
+  """edit module test.
+
+  Generally the tests are in two steps:
+  - modify an existing graph.
+  - then make sure it has the expected topology using the graph matcher.
+  """
+
+  def setUp(self):
+    """Set up."""
+    self.graph = ops.Graph()
+    with self.graph.as_default():
+      self.a = constant_op.constant([1., 1.], shape=[2], name="a")
+      with ops.name_scope("foo"):
+        self.b = constant_op.constant([2., 2.], shape=[2], name="b")
+        self.c = math_ops.add(self.a, self.b, name="c")
```

Comment:
If you simply import tensorflow, and use "tf.add" here; then it is much easier to read and maintain?